core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -528,8 +529,6 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     RemoveOtherReligions("Removes other religions when spreading religion", UniqueTarget.Unit),
 
     CanActionSeveralTimes("Can [action] [amount] times", UniqueTarget.Unit),
-    // TODO needs to be more general
-    BonusForUnitsInRadius("Bonus for units in 2 tile radius 15%", UniqueTarget.Unit),
```

Comment:
Instead of straight-up removing this, instead put this line above it:
```kotlin
@Deprecated("as of 4.1.0", ReplaceWith("[+15]% Strength bonus for [Military] units in [2] tiles"))
```

That way, modders get warnings they should update this unique. Additionally, when going to settings -> show mod errors -> a mod with this unique and press the `auto-update mods` button, it automatically replaces it in the mod itself, instead of the modder having to do it manually.